addons/hr_timesheet/controllers/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from . import portal
```

Comment:
Also, you should have a 'project.py' file extending the portal controller of project (method `_task_get_page_view_values`) and set the timesheet values (filtered with the portal domain). So, in template we should not use `task.timesheet_ids` in order to only display timesheet matching the domain.